code/modules/admin/sql_ban_system.dm

```diff
@@ -324,7 +324,6 @@
 				ROLE_REV_HEAD,
 				ROLE_SENTIENT_DISEASE,
 				ROLE_SPIDER,
-				ROLE_SWARMER,
```

Comment:
wrong one, from THIS list

Reply:
yes, you will break old bans if you don't

Reply:
are you sure? i thought this was just the list for the html page, like how you can click "antagonist positions" and auto select everything

Reply:
I am not sure

Reply:
@MrMelbert @jlsnow301 I aaam pretty sure you can remove all of these, the existing bans are in SQL anyway, so if swarmers get added back (theoretically) the bans will persist

Reply:
if you say so, then it's probably fine

Reply:
It's fine to remove from this list; existing role bans would only break if swarmers were added back with a different name for their define in `role_preference.dm`